lib/tjbot.js

```diff
@@ -841,9 +858,21 @@ TJBot.prototype.speak = function(message) {
         throw new Error("TJBot tried to speak a null message.");
     }
 
+    //get voice
+    var selectedVoice = "en-US_MichaelVoice"; //default voice if is not possible to find a better one
```

Comment:
i liked it better when this was defined up top with the rest of the default params, but since we got rid of that, i guess it needs to be defined here.